src/inmanta/server/config.py

```diff
@@ -183,6 +182,14 @@ def validate_fact_renew(value):
     is_list,
 )
 
+server_access_control_allow_origin = Option(
+    "server",
+    "access-control-allow-origin",
+    None,
+    "Configures the access_control_allow_origin setting of the http server."
+    "Defaults to not sending an AOCA header.",
```

Comment:
I prefer to use the full name here. If I google the term I do not find anything related to access control allow origin. I do find: "American Osteopathic College of Anesthesiologists (AOCA)"